baseapp/baseapp.go

```
@@ -782,6 +808,14 @@ func (app *BaseApp) deliverTx(tx []byte) *abci.ExecTxResult {
 		telemetry.SetGauge(float32(gInfo.GasWanted), "tx", "gas", "wanted")
 	}()
 
+	// Extract signers from the tx bytes
+	signers, signerErr := app.extractSigners(tx)
+	if signerErr != nil {
+		// Log the error but don't fail the tx
+		// For the sake of backwards compatibility??
+		app.logger.Error("failed to extract signers", "error", signerErr)
```

[not blocking] in general I think logging the error and proceeding is more likely to let a bug go unnoticed rather than propagating the error. So it may make sense to propagate the error instead of logging and continuing.

---

Happy to propagate the error the only reason I was not was to preserve backwards compatibility but I'm not sure if that's really an issue here